### ISSUE TITLE
Function request the end delivery rejected

### ISSUE THREAD

**Issue by NanShang126** [pop4959/ChunkyBorder#80]
Function request: When transmitting through refraction at the end, if the transmission exceeds the boundary, it should be rejected instead of being directed after transmission

**Comment by pop4959** [comment]
I have no clue what you're suggesting. Can you re-phrase / explain more?

**Comment by NanShang126** [comment]
For example, I set the world boundary to 500, but through the End Gateway Block, I can teleport beyond 500 and be directed to the boundary, which can cause the player to be in the void. Is it possible to reject transmission by detecting whether the End Gateway Block transmission point exceeds the boundary

**Comment by NanShang126** [comment]
![2.png](https://s2.loli.net/2024/08/29/avumxLnjbB3YIOk.png)
![1.png](https://s2.loli.net/2024/08/29/DYo5cCiaIgK3msM.png)

**Comment by pop4959** [comment]
Ah so you are saying this is a bug with end gateways not redirecting the teleport to a valid location?

**Comment by NanShang126** [comment]
No, this is a functional request. It will reject transmission if the transmission location exceeds the boundary, rather than transmitting to the vicinity of the boundary.
This is not that it was not directed to a valid location. My previous report showed that during the transmission, it should have been transmitted outside the boundary, but because the transmission point exceeded the boundary, it was transmitted to the vicinity of the boundary by ChunkyBorder

Sorry, my English is not good and I can only use translation software for my report

**Comment by pop4959** [comment]
So instead of teleporting the player do you want it to send a message and prevent the teleport?

You are asking for end gateways to be disabled instead of redirecting the teleport?

**Comment by NanShang126** [comment]
yes
